code-challenges.test.js

```diff
@@ -21,9 +23,44 @@ const fibLength1 = 6
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+   describe("fibonacciNyc", () => {
+
+    it(" Create a function name fibonacciNyc, fibonacciNyc will return array length of the numbers from the Fibonacci sequence, fibLength2 are the array length being passed through fibonacciNyc", () => {
+
+        expect(fibonacciNyc()).toEqual("[1, 1, 2, 3, 5, 8, 13, 21, 34, 55]")
+
+    })
+})
+
 
 // b) Create the function that makes the test pass.
 
+//---- I created a function name fibonacciNyc
+//---- fibonacciNyc will returned any arrays length of the numbers from the Fibonacci sequence 
+//---- When fibLength1 and fibLength2 are passed thru fibonacciNyc... I created if statements to create a looping 
+//---- process of the current sum being adding to the previous sum ang returned as an array
+
+const fibonacciNyc = () => {
+
+    nyc = [1, 1]
+
+    if (fibLength2 <= 1)
+    if (fibLength2 == 2){console.log(...nyc)}
+    if (fibLength2 > 2) {
+
+        for (let i = 1; i <= (fibLength2-2); i++) {
+            nyc.push(nyc[nyc.length-1] + nyc[nyc.length-2])
+        }
+        console.log(...nyc)
+    }
+}
+console.log(fibonacciNyc()) 
+
+//---- fibLength1 console.logged: [1, 1, 2, 3, 5, 8]
+//---- fibLength2 console.logged: [1 1 2 3 5 8 13 21 34 55]
+
+
```

Comment:
Great Job!  Your test and code are right on - they will give you the correct output. You just need to make them dynamic.  See notes for same issue on next two questions.  You're right there, Rick!  You are grasping the material, just need to focus on making everything dynamic.  Don't forget to use parameters! :)